modules/claims_api/app/workers/claims_api/vmbs_updater.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'bgs'
+
+module ClaimsApi
+  class # frozen_string_literal: true
+
+    require 'sidekiq'
+    require 'bgs'
+    
+    module ClaimsApi
+      class VBMSUpdater
+        include Sidekiq::Worker
+    
+        def perform(power_of_attorney_id)
+          poa_form = ClaimsApi::PowerOfAttorney.find(power_of_attorney_id)
+          service = BGS::Services.new(
+            external_uid: poa_form.external_uid,
+            external_key: poa_form.external_key
+          )
+          service.vet_record.update_birls_record(
+            participant_id: form_data.form_data['participant_vet_id'] || target_veteran.participant_id,
```

Comment:
I believe this is the right id in this case, but I've been burned on that before, so calling it out here for validation

Reply:
seems strange to use a Veteran id from the request payload i guess. should we not just use the the participant_id of the target_veteran always?

Reply:
yeah you're right, I don't even think we accept those arguments for POA typically, anyway. looks like just ITF. removing the form data